MediaBrowser.Providers/TV/SeriesMetadataService.cs

```diff
@@ -204,6 +204,14 @@ private async Task UpdateAndCreateSeasonsAsync(Series series, CancellationToken
             var seasonNames = series.SeasonNames;
             var seriesChildren = series.GetRecursiveChildren(i => i is Episode || i is Season);
             var seasons = seriesChildren.OfType<Season>().ToList();
+
+            // Very old Jellyfin library will have a cursed DB that a season may have NULL index number
```

Comment:
Are we not able to add a migration step to change the null index number?

Reply:
At this point, I don't know if we can. It's far too late in the release process, and anyone who's already tested unstable is affected (it's how I found this). Plus, I get the feeling fixing it in scanning is the right move anyways, but I'm not married to that opinion.

Reply:
I am not familiar with the scanner so I am fine with any solution that, someone who is familiar with it, sees as best fitting. I trust that decision. However, I am always a bit worried about implementing bug fixes with work around instead of fixing a root cause. We should avoid adding bug fix/ work around complexity to scanners, when it can be avoided. I also have a feeling that we might be jumping the gun here a bit (just a hunch I do not have a good grasp of the situation).

As for affected users. I personally see unstable as unstable and we should not be too reluctant to have breaking changes inside unstable versions. If something is discovered in the unstable builds, then it should be fixed accordingly, without being _too_ reluctant to break unstable setups. People should know the risk of running unstable and should treat it like so.

Reply:
> fixing a root cause

This is already impossible, can you invent Time machine so that we can go back to 2020 and fix that cursed scanner in Jellyfin 10.4?

Reply:
Then, in my opinion it should be part of a migration step. I am not familiar with this part of the code, but if all this is doing is making sure values are inserted during a scan, why not do that in a migration step so it runs only one time? As after that it wont be needed.